engine/function/src/templates/Basic.ftl

```diff
@@ -122,11 +123,12 @@ public class Basic {
      * @return array containing value, if value is not null according to Deephaven convention, replacement otherwise.
      */
     static public <T> T[] replaceIfNull(ObjectVector<T> values, T replacement) {
-        final int n = values.intSize("replaceIfNull");
         T[] result = values.toArray();
+        int i = 0;
 
-        for (int i = 0; i < n; i++) {
-            result[i] = replaceIfNull(values.get(i), replacement);
+        for (final T v : values) {
+            result[i] = replaceIfNull(v, replacement);
+            i++;
```

Comment:
You should revert this change, and instead replace just the unnecessary lookup in `values` with a lookup in `result`:
```
            result[i] = replaceIfNull(result[i], replacement);
```